kmath-histograms/src/commonTest/kotlin/space/kscience/kmath/histogram/UniformHistogram1DTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018-2021 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package space.kscience.kmath.histogram
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import space.kscience.kmath.distributions.NormalDistribution
+import space.kscience.kmath.misc.UnstableKMathAPI
+import space.kscience.kmath.operations.DoubleField
+import space.kscience.kmath.stat.RandomGenerator
+import space.kscience.kmath.stat.nextBuffer
+import kotlin.native.concurrent.ThreadLocal
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@OptIn(ExperimentalCoroutinesApi::class, UnstableKMathAPI::class)
```

Comment:
I think it is better to use `languageSettings.optIn(...)` for all test source sets instead of adding them in each test suite.

Reply:
Don't see much difference. This way we do not make build logic more complicated. Also we can stabilize featrures one by one.